lib/cisco_node_utils/bgp_neighbor_af.rb

```diff
@@ -170,7 +170,7 @@ def allowas_in
 
     def allowas_in_max
       val = allowas_in_get
-      val = default_allowas_in_max if val.nil? || val.zero? # workaround for CSCuv86255
+      val = default_allowas_in_max if val.nil? || val.zero? # CSCuv86255
```

Comment:
References to internal stories should probably be removed since they won't help external developers